apps/test/unit/code-studio/pd/workshop_dashboard/components/workshop_enrollment_pre_surveyTest.jsx

```diff
@@ -4,12 +4,14 @@ import {shallow} from 'enzyme';
 import {expect} from 'chai';
 
 describe("WorkshopEnrollmentPreSurvey", () => {
+  let fakeWorkshopDate = "October 4th";
+
   describe("getSortableUnitLessonShortName()", () => {
     let getSortableUnitLessonShortName;
 
     before(() => {
       getSortableUnitLessonShortName = shallow(
-        <WorkshopEnrollmentPreSurvey enrollments={[]}/>
+        <WorkshopEnrollmentPreSurvey enrollments={[]} workshopDate={fakeWorkshopDate}/>
```

Comment:
style nit: now that we're passing multiple props, let's expand this over multiple lines.

Reply:
See https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react